Dockerfile.www_server

```diff
@@ -0,0 +1,12 @@
+# Use const version of Ubuntu in case future versions won't be able to do sth from the bottom.
+FROM ubuntu:22.10
+RUN apt-get update
+RUN apt-get install -y apache2
+RUN apt-get install -y systemctl
+RUN apt-get install -y nano
+RUN apt-get install -y net-tools
```

Comment:
to samo co w poprzednim pliku

Reply:
Ta sama odpowiedź co w poprzednim. Ja myślę, że to nie jest tak bardzo więcej. Te kilka MB w te czy we wte, a daje duże możliwości w razie czego. Może będzie kazał nam coś sprawdzić.